docs/queries/filters.md

```diff
@@ -21,16 +21,30 @@ parent: Queries
 
 ## Searching for dates
 
-### Specific dates
+In Tasks query blocks, Tasks allows a lot of flexibility in the interpreting of dates.
 
-`<date>` filters can be given in natural language or in formal notation (`YYYY-MM-DD`).
-The following are some examples of valid `<date>` filters as inspiration:
+### Absolute dates
 
-Formal notation:
+`<date>` filters can be given with 'absolute' dates, whose preferred format is `YYYY-MM-DD`.
+
+Absolute dates specify a **particular date in a calendar**. They represent the same day, regardless of today's date.
+
+Example absolute dates:
```

Comment:
We are already in the absolute dates sections, I guess `Examples:` will be enough =)

Reply:
> We are already in the absolute dates sections, I guess `Examples:` will be enough =)

Done.